api/soap/mantisconnect.wsdl

```diff
@@ -45,6 +45,7 @@
    <xsd:element name="account_data" type="tns:AccountData" minOccurs="0"/>
    <xsd:element name="access_level" type="xsd:integer" minOccurs="0"/>
    <xsd:element name="timezone" type="xsd:string" minOccurs="0"/>
+   <xsd:element name="api_token_type" type="xsd:integer" minOccurs="0"/>
```

Comment:
It is not strictly necessary, but desirable. 

Here is an usage case: imagine if you develop application. You don't want to reinvent the bicycle, so instead of writing your own error reporting solution, you pick up some 3rd party software. So, you feed you login credentials and other settings into software. Now, your application crashes and this software has to send bug report to Mantis. Currently this software would make a search for "fake alias", made insert or update. If this pull request will be implemented, then this software could simply use new report-only API or it could stick to old method, if there are some additional logic involved. 

How does this software recognize if it could call other methods from API? Try to call it and fail when access denied is returned? It could. But it also could just determinate what type of token you (as developer of application which uses this software) have provided. Type would determinate which API calls software can make.